frontends/web/src/routes/settings/components/mobile-header.module.css

```diff
@@ -35,10 +35,14 @@
     font-weight: 400;
     margin: 0 auto;
     position: absolute;
-    left: calc(50% + 16px);
```

Comment:
just out of curiosity does this fix the horizontally centering of the settings title?